server/text_generation_server/models/custom_modeling/llava_next.py

```diff
@@ -136,7 +136,7 @@ def __init__(self, prefix, config, weights):
         self.config = config
         config.text_config.quantize = config.quantize
         config.text_config.speculator = config.speculator
-        self.language_model = load_text_model(
+        self.text_model = load_text_model(
```

Comment:
Why?

Reply:
So it's uniform and `vlm_causal_lm` works consistently for both.